packages/web/src/components/search/addons/Mic.js

```diff
@@ -129,7 +129,9 @@ class Mic extends React.Component {
 		const { className } = this.props;
 		switch (status) {
 			case STATUS.active:
-				return <ListenSvg className={className} onClick={this.handleClick} />;
+				return (
```

Comment:
@mohdashraf010897 It seems like a formatting issue.

Reply:
not a formatting issue
https://www.loom.com/share/0f0b5c9a1f494957951237e9222d0e94